runelite-client/src/main/java/net/runelite/client/plugins/bank/BankConfig.java

```diff
@@ -23,14 +23,14 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.bankvalue;
+package net.runelite.client.plugins.bank;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
-@ConfigGroup("bankvalue")
-public interface BankValueConfig extends Config
+@ConfigGroup("bank")
```

Comment:
Won't this cause existing configs to be forgotten? Seems like we should migrate those.

Reply:
I dont think it matters at all